### ISSUE TITLE
Q&A w/ @zkat on Individual Membership Candidacy

### ISSUE THREAD

**Issue by hackygolucky** [nodejs/board#44]
This thread is for asking @zkat questions regarding [her run](https://gist.github.com/zkat/345d1485fc4cd1f45155678a3729cd21) for the Node.js Foundation Board of Directors.

**Comment by nebrius** [comment]
Hi @zkat,

Thank you for running for the Node Foundation community seat! To help me and others get to know you better, I have a set of questions that I'm asking all candidates, listed below. I look forward to your answers!

1) How did you get involved in the Node Project?

1) What does inclusivity and diversity within the Node project look like to you? How would you work with the board to increase inclusivity and diversity within the Node project?

1) How would you work with the board to increase international participation in the Node project?

1) What do you think are the pros/cons about the separation of powers/responsibility between the TSC/CC and the Board? How would you improve it?

1) What do you think is the greatest weakness in the Node Project? How would you work with the board to fix it?

1) What do you think is the greatest strength in the Node Project? How would you work with the board to preserve it?

**Comment by jasnell** [comment]
Just two quick questions from me: (a) In your view, what is the primary role of the Board in the evolution of the Node.js platform and (b) as an individual board member, what would be one thing you'd hope to accomplish over the next year?

**Comment by zkat** [comment]
@nebrius 

1. My main avenue into getting involved with the Node project proper was through my employer, npm. As one of the devs for the CLI, it kinda put me in a position to interact with the Node community a lot, including sending new versions of npm over, coordinating with LTS, speaking at conferences, etc.

2. What I really want with respect to this is to have systems in place such that folks who are underrepresented in the tech community are able to participate comfortably and openly without having to hide or sacrifice aspects of who they are. The fact is, yes, there's simply fewer `X` people in the tech community. I think getting that number closer to actual demographic representation is nice, but I think the most important part is to make sure that if there happens to be someone already here, or someone actively interested in joining, that they're not scared off by what they see or hear. That's something we as a community have much more control over as well: we can put things in place that will get folks to say "this place? Node? it's alright", even if we can't guarantee that we'll actually get interest from this or that population. This is kind of a long-winded way of saying that the "diversity" that a lot of people perceive as just being "put some quotas in there and force the matter" are less interesting to me than putting processes in place to make sure we don't have random unpleasant barriers (though actual representation is a way for voices to speak up about needs). One approach I'm thinking of taking through the boards is to use the position to raise awareness within the overall project -- I mentioned a survey (which apparently is already in the works!) that I think is an excellent first step to going forward with this. I want to have concrete data to show the Node community. I want that data to be public. I want accountability both at the board and Node Core when it comes to ways the system might be failing those underrepresented folks. I would have relatively little (read: no) control over actual Core project policy, but I can definitely push for improvements surrounding Foundation-funded events and initiatives that Core intends to benefit from. The rest is a matter of coordinating with the rest of the board to see what else is possible, and looking at data from that survey to come up with more concrete, targeted approaches that address the real weak spots.

3. I am by far more concerned about the lack of representation in countries the Node project already draws participants from than about expanding international participation further. I do have contact with some international communities (some folks in Japan, Brazil, and Mexico, and I'm getting more active in the Spain/Barcelona tech community), but it's simply not my priority in any measurable way right now. Bluntly, I have felt in the past that "international participation" has been a red herring in inclusivity discussions to distract from issues around URMs (underrepresented minorities). To be clear: increasing representation of cis men from the dominant ethnosocial group of a variety of countries is not a very strong gesture of inclusion. Increasing language support for dominant sociolinguistic groups is not a very strong gesture of inclusion. And therefore, it will not be my priority when it comes to inclusivity issues.

4. My brief experience with this divide was frustrating: it felt as though it was better at keeping things in a standstill than actually being of use. I see the primary reason for this divide existing to have been a desire to separate certain business interests from technical decision making. When it came to the Inclusivity Working Group, though, these issues went beyond simple technical decisions (presumably putting them more in the hands of the Board), yet involved decisions on technical governance (putting them in the hands of the then-TSC, which had a bit of an identity crisis itself). I do not think a full hard divide between these two sides of Node governance is a good idea when it comes to social issues like these: I'd like there to be more direct cooperation between the ones who sign the checks and the ones who manage the github issues. I think it's the only functional way of moving forward with this. We shouldn't have to be doing a back-and-forth dance through a bureaucratic swamp any time the issue at hand is anything other than who gets to decide whether a PR to the C++ code gets merged.

5. There are capable people who tell me they feel excluded from the project because of who they are. Talented individuals who shy away from the reputation Node has developed, in spite of its efforts to be a strong community-driven project. The myth of the meritocracy is deeply entrenched in the ethos of many of its members, which involves two harmful illusions: A) that anyone who is technically capable is able to participate and succeed; B) anyone who has succeeded and climbed up the social "ranks" within the community has done so due to greater technical merit than their peers. Both of these statements are entirely false and I think Node would do well with working towards extricating them wholesale from its community values.

6. Node.js is a strong and growing technology. Its growth and market success have facilitated millions of users in professional environments. There's strong interest both from the wider open source community and from businesses in making the most of this technology, and thus in helping it succeed further. I think preserving this growth is important because of the opportunities it will create as more and more jobs become available. I have no concrete suggestions for how to "preserve" it since I think pretty much any initiative the Board will take will be directed towards this goal, whether that be running conferences, promoting business connections with larger enterprises (while preserving Node's current "hot stuff" reputation among smaller businesses, one would hope), or funding education of new Node developers to fill in all those new openings in the job market.

@jasnell:

a. In the most general sense, promoting the growth and success of the platform shared by its participating (and often competing) members. This tends to manifest in terms of business connections, funding of events, and community-related initiatives like Node Together. The more folks learn about Node and the more folks are able to successfully use it, the better the Board is doing its job.

b. I outlined [in my announcement](https://gist.github.com/zkat/345d1485fc4cd1f45155678a3729cd21) my plans to push for a community survey that can help us get a better sense of who our users are (and who they _aren't_ right now). @hackygolucky told me afterwards that this is already an initiative, so I plan on fully supporting that and doing my best to make it successful and useful. On a larger scale, I want more, different people to openly give feedback on how much Node is improving when it comes to supporting underrepresented folks. We've seen similar shifts in public reputation from companies like Github, for example. And I want to be able to have to tools to measure this change so we're not just going by some gut feeling about community sentiment. Let's have the data to back it all up.

**Comment by bnoordhuis** [comment]
> Bluntly, I have felt in the past that "international participation" has been a red herring in inclusivity discussions to distract from issues around URMs (underrepresented minorities). To be clear: increasing representation of cis men from the dominant ethnosocial group of a variety of countries is not a very strong gesture of inclusion. Increasing language support for dominant sociolinguistic groups is not a very strong gesture of inclusion. And therefore, it will not be my priority when it comes to inclusivity issues.

Wow, that's an appallingly US-centric viewpoint.

How do you feel about the fact that we have almost no participants from the African continent even though it's home to 1/6th of the world population?

I hope I'm wrong but, going by your statement, you essentially say "don't care, not interested."

**Comment by zkat** [comment]
@bnoordhuis Seems to be a skip and a hop from "not my priority" to "don't care, not interested". There doesn't need to be exactly one thing I need to do, but let me outline some priorities to help clarify my viewpoint a bit. In order of most important to least, for me:

1. Underrepresented minorities, and especially People of Color and their intersections, in countries included in the primary economies (US, Europe, Japan, Australia, etc), which I believe involve the majority of current contributors to Core. Considering how many folks we have from those countries, it seems important to make sure that we're at least doing a good job on this in countries already well represented.
2. Underrepresented minorities in countries that we don't already have significant developer representation in (see next point)
3. Other folks from non-"primary" economies (no, I don't know what the latest best term for "not superpower economies" is these days, but if I should correct this, lmk!). That is -- Latin America, Africa, South/East/Southeast Asia and folks in the Pacific, etc.
4. Literally everyone else (Folks from the US, Canada, Australia, and Europe. Possibly Japan)

If we can't serve URMs in the countries we already have, it seems like a stretch to imagine we'd do anything like a good job for countries that have even lower overall representation.

I think you might be missing some of the context under which I emphasize that privileged folks from underrepresented *nations* are less interested to me. I'm Puerto Rican. I grew up on an island in the Caribbean as a fairly privileged, white-for-my-country, upper-middle-class person. English is not my first language. I've had the chance, since I moved to the US, to gain a lot more context around how privilege works. I've particularly learned the difference between the oppression URMs suffer when they've grown up in a country that oppressed them, vs coming in as an immigrant -- and I've heard similar experiences from folks from African countries that make it to the US and view its racism from a completely different light (as I once did).

So, tl;dr: Having more African participants is great. I will prioritize their comfort over yours, @bnoordhuis. But folks who are marginalized in their own countries will be my priority.

**Comment by jasnell** [comment]
I certainly do not interpret @zkat's statement as saying "don't care". We each bring a personal perspective into these discussions and each of us focuses on the areas that we view as needing the most attention. It's certainly not a zero sum game by any measure. I am quite happy to see continuing effort going into not only growing the community but in making it a more welcoming place for everyone. Thank you @zkat for your responses and for your willingness to continue working with the project and foundation.

**Comment by bnoordhuis** [comment]
@zkat Appreciate the thoughtful reply.

**Comment by mcollina** [comment]
@zkat thanks for providing such good points for a heated conversation. I think it is good for the community.

 What do you mean when you said 

>"international participation" has been a red herring in inclusivity discussions to distract from issues around URMs"

There was some underlining discussions that your brought into this discussion, without exactly providing a summary of what happened and what you want to change there.

> The myth of the meritocracy is deeply entrenched in the ethos of many of its members, which involves two harmful illusions: A) that anyone who is technically capable is able to participate and succeed; B) anyone who has succeeded and climbed up the social "ranks" within the community has done so due to greater technical merit than their peers. Both of these statements are entirely false and I think Node would do well with working towards extricating them wholesale from its community values.

Can you please clarify why you think those statements are false? Those ring strage to me, but I might had a different experience than a lot of other people, and I would like to know more and help.

**Comment by zkat** [comment]
@mcollina when I'd been an npm for a couple of months, I was invited to be part of the Node.js Collaborator Summit in SF. At that event, which was two days long, I was the only woman out of 40+ people. By US standards for the definition of People of Color (namely: someone belonging to a marginalized ethnicity in their current country), I was also one of few I recall.

One of the topics that we brought up was inclusivity/diversity. I have a very vivid memory in my mind of one *very prominent* member of the community making their sole, firm contribution being that we didn't have an inclusion problem because anyone who wanted to contribute, could. They then proceeded to insist that, instead, we should focus on making it easier for more people from different timezones to contribute.

This flies in the face of the situation surrounding him: one woman in the room, few who weren't literally US/Canada/Europe white cis dudes. And what I consider a blatant denial of the experiences I have heard from various folks in the community about feeling excluded/scared to contribute/turned off by this or that behavior by contributors.

Since then, I've heard mention of "international participation" being used as a soft derail to these conversations: @bnoordhuis' commentary here read to me to be just along those lines. Perhaps I'm being a bit sensitive about this here, and I can certainly not make assertions about others' intentions. I do not believe this is the opinion of even a majority of Node contributors. Quite the opposite: I've heard a lot of support for inclusion and diversity being voiced at all levels of the organization, even if I disagree with their particular approaches. I just think we need to make a clear statement about what our priorities are, what we mean, etc. I tend to avoid the terms "inclusivity and diversity" in many conversations because of this. I have a very specific idea of what that means and what priorities I will have for that.

To your second question: I believe that these statements are both false regardless of any improvements we ever make with this community. The problem here is not that people are being "bad" and preventing that from being true. The problem is that the reality of our world is that meritocracy simply sweeps issues around oppression under the rug. Yes, marginalized folks are able to perform just like anyone else at a technical level, but they will, on average, experience many more hurdles. Their talents will be scrutinized more. Their successes will be celebrated less. They will have to speak louder just to be heard at all. There is literal research on the various often-subconscious effects that are in place in any given situation that involve sexist, racist, etc behaviors. And even people that have very consciously chosen to break through them will find it challenging.

So, I think it's important to let go of hoping to "simply" set up a system where people can "just" succeed based on their technical merit. We need to be aware of the oppressions of the system we exist in. We need to actively and proactively maneuver around and compensate for the very real systems of oppression that many of our contributors have experienced. Treating everyone the same will invariably only benefit those who have had the privilege to live their life with fewer hurdles.

**Comment by bnoordhuis** [comment]
> They then proceeded to insist that, instead, we should focus on making it easier for more people from different timezones to contribute.

I wasn't at the SF summit but that sounds pretty sensible to me.  Most day-to-day activity is scheduled around US office hours (case in point: it's almost midnight local time and I'm still at work); the long communication delays, to say nothing of scheduling meetings, is pretty off-putting to people from Russia, India, Southeast Asia.

You seem to find that offensive but why?  Different definitions of 'inclusive'?

**Comment by zkat** [comment]
@bnoordhuis To be clear: I've run into international scheduling stuff and its related frustrations, and the obnoxious Bay Area-centereredness of it all. Yeah, it sucks.

Like I outlined above, this is a matter of my priorities, though: I want to know that for folks in the timezones that we're already biased against, we're serving folks already. I think if you think of it in terms of expansion, we'll be much better equipped to handle the complexities of expanding across languages, timezones, and cultures, if we take advantage of the richness of variety that could be tapped for ideas and talent in the languages, timezones, and cultures that we already have an easy time supporting. They may very well open doors into those other timezones.

The key thing here is that expanding internationally is much more than translations and setting meeting times which don't force people to be up at 3am: it requires a concerted effort to make those who feel like they can't fit in, as themselves, know that they'll be welcomed with open arms and provided with the resources they need.

You mentioned "US-centric" before. I'm arguing that there's few things more US- and Euro-centric than assuming that a community shaped by and around privileged groups in those two centers can actually serve the needs of *anyone* outside them without serious adaptation.

But let's talk about this a bit more with a couple of specifics: As a member of the CLI team, I've had the privilege to have communicated directly with members of these international communities on a regular basis. My gut feeling is that those interactions have possibly been more numerous and even more detailed than those experienced in github.com/nodejs/node -- because they're often support tickets.

The most startling thing for me has been how many Chinese and Indian users we have, and how utterly disconnected from our community they seem. Most Chinese users operate against cnpmjs.org, which has a mirror that sits behind the Great firewall. They also seem to have a fairly hard time with English, and that's made communication with my team pretty hard at times, but I'm glad they still reach out to us, and they're generally kind, helpful people trying to get their work done. Indian users I've run into are similar, and even have fairly similar technical constraints: they're almost always behind some bizarro corporate firewall, and they reach out to my team because really they just want to get their work done and something has gotten in the way of that.

I've lamented in the past about how little participation from Indian developers participate in our community. It's also a thing I've casually grumbled about while adminning for wealljs.org -- we have a few Indian folks there, some being very active, awesome contributors, but I'm often at a loss about whether we're fulfilling their needs. I'd love to find ways to help them engage more. There's certainly a lot of talent there. The same goes for Chinese developers even if part of me is at a loss about how to effectively get through the social issues the Firewall can create. Not that I think it's hopeless, though, but I haven't had `Infinity` time to think through all these bits in detail.

I *have* been doing more active outreach around wealljs with Brazilian and Spanish tech communities, to name specific ones. I've gotten to know a number of prominent folks from both of those, joined their own online communities, and invited them into mine, and they've invariably enriched the experience for everyone involved. I'm also kinda by-default in contact with some Caribbean tech folks (specially the diaspora) purely from belonging to that demographic myself, though I'd like to have more solid connections there.

And as far as Russians go? Well, I've on multiple occasions had to remind admins in communities I help with about the Slavic impatience for roundabout, indirect, "polite" speech (in US and UK terms), and their desire to be clear, direct, and blunt in ways that really rubs some US folks the wrong way. Like I said, it's more than timezones. We have to have the resources to understand how to engage folks on their terms rather than pull them into whatever culture we've created by default.

Sorry if that's a lot, but there's a lot to say, and I think it would take much longer to take all these bits and turn them into concrete proposals. I hope it at least drives the point home that I've not been advocating for ignoring international participation. We can do more than one thing just fine. I just wanna make it a priority to give attention to a very important thing I believe has been largely unattended.

**Comment by zkat** [comment]
Or, maybe what you were asking is why I found that specific statement offensive: I found it offensive because in a context that I considered tremendously non-inclusive (because actual participation was very homogenous), representation by folks like me, and other folks who simply weren't there at all, was largely ignored, in favor of "timezones".

**Comment by Fishrock123** [comment]
Two questions (for each of the candidates):

@zkat:

1. How will you communicate with broader Node community you could be representing? What would be the primary avenue for questions and comments to yourself and to the board through yourself?
2. How would make future Individual Directors more effective, even continuing past the current term?

**Comment by zkat** [comment]
@Fishrock123 

1. This is really really really really hard. This is a question that @othiym23, @iarna, and I would be constantly discussing because we just couldn't find any clear mechanism that actually communicated across the massive, wide community of npm users. Pretty much our main means was through the issue tracker whenever they took the initiative to post stuff, assuming they didn't abandon them before we could really engage. Twitter, IRC, Slack, etc, all proved to be fairly poor mediums at actually reaching out to people, specially when it involved pushing new information out to them. We've kinda settled on using our newsletter as the main mechanism for outreach, but this is a very limited medium. I would love to work with the Board to find more effective means of actual outreach -- be it surveys, local events, hangouts, or literally anything else we could try and the Board could fund. Hopefully something will stick and we'll learn a few things. It's pretty gnarly that I'll give a talk somewhere and remind folks that 40% of our userbase is using npm from Windows, and they'll be super surprised. There's a huge chasm of communication there, when we don't really talk to such large swaths of our users. :\

2. Maybe it's getting late or there's something with the grammar for this one or I'm missing some context, but ENOPARSE. Do you mind if I ask you to reword this question and maybe provide a little more context? Do you mean Individual Directors continuing to be effective once their own terms have expired, or setting things now that will benefit other Directors once my own term has expired?

**Comment by bnoordhuis** [comment]
@zkat Great reply, thanks.  I understand where you're coming from a bit better now.

(For people reading along: that's in reply to https://github.com/nodejs/membership/issues/44#issuecomment-274965711.)

**Comment by mcollina** [comment]
Thanks so much for the clarification @zkat!
